src/domain/session/room/RoomViewModel.js

```diff
@@ -143,14 +141,8 @@ export class RoomViewModel extends ViewModel {
     get timelineViewModel() { return this._timelineVM; }
     get isEncrypted() { return this._room.isEncrypted; }
 
-    get error() {
-        if (this._timelineError) {
-            return `Something went wrong loading the timeline: ${this._timelineError.message}`;
-        }
-        if (this._sendError) {
-            return `Something went wrong sending your message: ${this._sendError.message}`;
-        }
-        return "";
+    get errorViewModel() {
+        return this._errorViewModel;
     }
```

Comment:
Do we not already inherit this getter from `ErrorReportViewModel`?

Reply:
We do! good catch